venter/serializers.py

```diff
@@ -33,14 +33,15 @@ class Meta:
         model = Complaints
         fields = (
             'id', 'created_by', 'description', 'report_date', 'status', 'latitude',
-            'longitude', 'location_description', 'tags', 'comments', 'users_up_voted', 'images'
+            'longitude', 'location_description', 'tags', 'comments', 'authorities', 'users_up_voted', 'images'
         )
 
 class ComplaintPostSerializer(serializers.ModelSerializer):
     class Meta:
         model = Complaints
         fields = (
-            'id', 'description', 'report_date', 'latitude', 'longitude', 'location_description')
+            'id', 'description', 'report_date', 'latitude', 'longitude', 'location_description'
+        )
```

Comment:
Whitespace change

Reply:
You need to fix this

Reply:
should i just revert? the whitespace given in the modification is how it would look with standard indentation, if we wrote it in the accepted style

Reply:
Reverted